src/mixins/filing-template-mixin.ts

```diff
@@ -536,6 +543,15 @@ export default class FilingTemplateMixin extends DateMixin {
     // restore Persons and Organizations
     this.setOrgPersonList(this.partiesToOrgPersons(draftFiling.restoration.parties || []))
 
+    // restore Office addresses
+    this.setOfficeAddresses(draftFiling.restoration.offices)
```

Comment:
Are office addresses guaranteed to be in the draft filing? If so then this is OK. If not then you should provide a suitable fallback value.

Reply:
I think you should have a check here, otherwise you could set office addresses to null or undefined. (And undefined is bad because the property is no longer reactive if it has that value.)

Reply:
Just thinking I missed something from our discussion. Adding it.

Reply:
Done